### PR TITLE
feat!: Drop `url_config` column from `aws_lambda_function_aliases` table

### PR DESCRIPTION
**Pull request by candiduslynx**
Follow-up for https://github.com/cloudquery/cloudquery/pull/14913
Part of https://github.com/cloudquery/cloudquery/issues/12648

BEGIN_COMMIT_OVERRIDE
feat: Drop `url_config` column from `aws_lambda_function_aliases` table (#14997)
BREAKING-CHANGE: Drop `url_config` column from `aws_lambda_function_aliases` table (#14997). The data now resides in a separate `aws_lambda_function_url_configs` table.

feat: Remove `function_url` column from `aws_lambda_function_url_configs` table (#14997)
BREAKING-CHANGE: Remove `function_url` column from `aws_lambda_function_url_configs` table (#14997). `function_arn` is sufficient as it includes the alias name as well (if the URL config fetched belongs to the function alias).
END_COMMIT_OVERRIDE

### CODE REVIEW

File: plugins/source/aws/resources/services/lambda/function_aliases.go [hunk 1]
```diff
@@ -7,19 +7,17 @@ import (
...
 )
 
 func functionAliases() *schema.Table {
 	tableName := "aws_lambda_function_aliases"
 	return &schema.Table{
-		Name:                tableName,
-		Description:         `https://docs.aws.amazon.com/lambda/latest/dg/API_AliasConfiguration.html`,
-		Resolver:            fetchLambdaFunctionAliases,
-		PreResourceResolver: getFunctionAliasURLConfig,
-		Transform:           transformers.TransformWithStruct(&models.AliasWrapper{}, transformers.WithUnwrapAllEmbeddedStructs()),
```
Comment (disq) [inline plugins/source/aws/resources/services/lambda/function_aliases.go:22]: You should also remove `models.AliasWrapper` and `models.RuntimeWrapper`

Comment (disq) [reply plugins/source/aws/resources/services/lambda/function_aliases.go:22]: which you did.

**Review by disq — APPROVED** [review]

**Comment by bbernays** [comment]
This PR seems to be completely removing data `url_config`. How are we going to make sure users can still access this data?

**Comment by candiduslynx** [comment]
> #14913

See https://github.com/cloudquery/cloudquery/pull/14913

**Comment by candiduslynx** [comment]
> This PR seems to be completely removing data `url_config`. How are we going to make sure users can still access this data?

See https://github.com/cloudquery/cloudquery/pull/14913

**Comment by bbernays** [comment]
That is only for `url_config` for function, if there is an alias it would have a different config.

**Comment by candiduslynx** [comment]
> That is only for `url_config` for function, if there is an alias it would have a different config. See how the input takes a qualifier:
> 
> ```
> 		Qualifier:    alias.Name,
> ```

I've tested this in our org, the list for the func will return both own cfg & alias, like so:
```sh
aws lambda get-function-url-config --function-name arn:aws:lambda:us-east-1:123456:function:test-func
```
will return
```json
{
    "FunctionUrlConfigs": [
        {
            "FunctionUrl": "https://bde.lambda-url.us-east-1.on.aws/",
            "FunctionArn": "arn:aws:lambda:us-east-1:123456:function:test-func:test-alias",
            "CreationTime": "2023-10-26T12:43:13.722212Z",
            "LastModifiedTime": "2023-10-26T12:43:13.722212Z",
            "AuthType": "AWS_IAM"
        },
        {
            "FunctionUrl": "https://abc.lambda-url.us-east-1.on.aws/",
            "FunctionArn": "arn:aws:lambda:us-east-1:123456:function:test-func",
            "CreationTime": "2023-10-26T12:43:34.064447Z",
            "LastModifiedTime": "2023-10-26T12:43:34.064447Z",
            "AuthType": "AWS_IAM"
        }
    ]
}
```

**Comment by candiduslynx** [comment]
Will reopen to [`feat/aws/v23-quality-of-life`](https://github.com/cloudquery/cloudquery/tree/feat/aws/v23-quality-of-life) branch

**Comment by candiduslynx** [comment]
Reopened as https://github.com/cloudquery/cloudquery/pull/15380

**Pull request closed without merging**